src/main/java/me/ryanhamshire/GriefPrevention/BlockEventHandler.java

```diff
@@ -232,6 +233,22 @@ public void onBlockPlace(BlockPlaceEvent placeEvent)
 		String noBuildReason = GriefPrevention.instance.allowBuild(player, block.getLocation(), block.getType());
 		if(noBuildReason != null)
 		{
+			// Allow players with container trust to place books in lecterns
+			PlayerData playerData = this.dataStore.getPlayerData(player.getUniqueId());
+			Claim claim = this.dataStore.getClaimAt(block.getLocation(), true, playerData.lastClaim);
+			if (block.getType() == Material.LECTERN && placeEvent.getBlockReplacedState() instanceof Lectern)
+			{
+				if (claim.allowContainers(player) != null)
```

Comment:
Should check if claim is not null here, and can set PlayerData#lastClaim if so.

Reply:
Good call, I dont know why I keep forgetting to check for claim being null.